private/dev/index.js

```diff
@@ -1,12 +1,43 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import Form from '@uppy/form'
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import Dashboard from '@uppy/dashboard'
+
 // eslint-disable-next-line import/no-extraneous-dependencies
 import 'uppy/src/style.scss'
 
 import DragDrop from './DragDrop.js'
-import Dashboard from './Dashboard.js'
+import DashboardUppy from './Dashboard.js'
+
 
 switch (window.location.pathname.toLowerCase()) {
   case '/':
-  case '/dashboard.html': Dashboard(); break
+  case '/dashboard.html': {
+    const restrictions = undefined
+    // const restrictions = { requiredMetaFields: ['caption'], maxNumberOfFiles: 3 }
+
+    const uppy = DashboardUppy({ restrictions })
+      .use(Dashboard, {
+        trigger: '#pick-files',
+        // inline: true,
+        target: '.foo',
+        metaFields: [
+          { id: 'license', name: 'License', placeholder: 'specify license' },
+          { id: 'caption', name: 'Caption', placeholder: 'add caption' },
+        ],
+        showProgressDetails: true,
+        proudlyDisplayPoweredByUppy: true,
+        note: `${JSON.stringify(restrictions)}`,
+      })
+      .use(Form, { target: '#upload-form' })
+
+    window.uppy = uppy
+
+    const modalTrigger = document.querySelector('#pick-files')
+    if (modalTrigger) modalTrigger.click()
+  }
+  break
```

Comment:
Why define uppy and its plugins both in index and the route files? I expect this to only link to the app file, not define it in two places.

Reply:
what do you mean by the route files?

I wanted to reuse uppy initialisation with all dev env variables and logic between react (React.jsx) and non-react (index.js). but for react we cannot `uppy.use(Dashboard)` so I pulled that out and instead do it only here in non-react (index.js). (and modalTrigger is not appliccable for react either, as well as `window.uppy = uppy`).

Reply:
I think this should be rewritten to have all config in it's place and not spread between two files